web/static/js/components/idea_submission_form.jsx

```diff
@@ -87,20 +117,8 @@ export class IdeaSubmissionForm extends Component {
       <form onSubmit={this.handleSubmit} className="ui form">
         {pointingLabel}
         <div className={`${styles.fields} fields`}>
-          <div className={`${styles.flex} five wide inline field`}>
-            <label htmlFor="category">Category:</label>
-            <select
-              id="category"
-              name="category"
-              value={this.state.category}
-              className={`ui dropdown ${styles.select}`}
-              onChange={this.handleCategoryChange}
-            >
-              { this.props.showActionItem ? <option value="action-item">action-item</option> :
-                defaultCategoryOptions
-              }
-            </select>
-          </div>
+          {stage === IDEA_GENERATION && <SelectDropdown {...dropdownProps} />}
```

Comment:
rather than repeat the stage condition checks from up above (see lines 90 - 98), you could DRY this up by setting a let declaration to `let dropdownProps` and, here, saying `dropdownProps && <SelectDropdown {...dropdownProps} />`

Reply:
done